vertical_flatten.py

```diff
@@ -105,8 +105,8 @@ def validate_for_another_schema(self, list_to_verify):
  "restaurant_id" : "30075445"
 }
 
-vf = VerticalFalten()
-print(vf.vertical_flatten_call(sample_document, return_as_dict=1))
+# vf = VerticalFlatten()
+# print(vf.vertical_flatten_call(sample_document, return_as_dict=0))
 
 # [{"Quality_Food": 4, "Safety": 4, "Quantity_Food": 5}, {"Quality_Food": 4, "Safety": 4, "Quantity_Food": 5}]
```

Comment:
Remove these comments also, its fine you can do this later as well. Otherwise looks fine